pkglistgen.py

```diff
@@ -1368,10 +1380,11 @@ def update_and_solve_target(self, api, target_project, target_config, main_repo,
         delete_products = target_config.get('pkglistgen-delete-products', '').split(' ')
         self.tool.unlink_list(product_dir, delete_products)
 
-        print('-> product service')
-        for product_file in glob.glob(os.path.join(product_dir, '*.product')):
-            print(subprocess.check_output(
-                [PRODUCT_SERVICE, product_file, product_dir, opts.project]))
+        if not product_converter_enabled:
```

Comment:
as this now is the 4th level of indentation, can you please extract this as a function (not to be called for product_converter_enabled)

Reply:
done